{{ cookiecutter.format }}/gradle.properties

```diff
@@ -19,3 +19,5 @@ android.useAndroidX=true
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+# Suppress warnings about Android SDK 34 compatibility
+android.suppressUnsupportedCompileSdk=34
```

Comment:
This can be removed if the Gradle and Gradle plugin versions are updated as suggested above.